README.md

```diff
@@ -1 +1,172 @@
-# Hello
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** is a simple website that is made out of HTML and CSS. It dicplays the text "Hello Microverse!" on the screen, colored cyan in css.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- HTML
+- CSS
+- HTML, CSS Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- A live demo for this project will be available once the project has been completed and deployed. It is still in progress.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly as stated by the previous reviewer 😃 try to remove the section of the live demo if you don't have the link of the demo right now this is for professional readme and when you learn how to deploy the project you can put this section again and put the link inside it 😊

| Requirements | 
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/226843303-5b8c2ad5-bb1b-4eba-bce5-28f0c681f89c.png)   |

Reply:
@osamaashraf6  Please check, I have made corrections.